runtime/monotouch-debug.m

```diff
@@ -537,13 +537,15 @@ void monotouch_configure_debugging ()
 		return;
 	}
  
-	// If debugging is enabled
+ #if !(TARGET_OS_MACCATALYST || TARGET_OS_OSX)
+	// If debugging is enabled (only check for mobile builds - for macOS / Mac Catalyst debugging is always enabled in debug versions of libxamarin)
```

Comment:
title say it's not .net/Mac[catalyst] only
but it seems to apply to current/legacy too
That should be fixed (title) and documented (in the release notes at least)

Reply:
This file (monotouch-debug.m) isn't included in legacy Xamarin.Mac, so this will not have any effect on legacy code.